Assets/Scripts/EOSFriendsManager.cs

```diff
@@ -436,21 +436,24 @@ private void OnQueryPresenceCompleted(ref QueryPresenceCallbackInfo data)
                 TargetUserId = data.TargetUserId
             };
 
-            Result joinInfoResult = PresenceHandle.GetJoinInfo(ref joinInfoOptions, out Utf8String joinInfo);
-            if (joinInfoResult != Result.Success)
-            {
-                joinInfo = null;
-            }
-
+            // If the user has a Presence JoinInfo, set that in this incoming Presence.
+            // NotFound just implies the user does not have an active joinable activity, it is not an error.
```

Comment:
It's unclear to me in this context what `NotFound` is referring to.

Reply:
`Result.NotFound`, a possible result of the method coming after that comment. I think you expressing confusion makes it clear that this comment should be elaborated a bit, and moved to where it actually belongs.